bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -243,54 +170,115 @@ outdir = os.path.split(os.path.abspath(opts.output_file))[0]
 if not os.path.isdir(outdir):
     os.makedirs(outdir)
 
-# Extract IFOs and vetoes
-ifos, vetoes = ppu.extract_ifos_and_vetoes(trig_file, opts.veto_files,
-                                           opts.veto_category)
-
-# Exit gracefully if the requested IFO is not available
-if ifo and ifo not in ifos:
-    err_msg = "The IFO selected with --ifo is unavailable in the data."
-    raise RuntimeError(err_msg)
+# Extract IFOs
+ifos = ppu.extract_ifos(trig_file)
+
+# Generate time-slides dictionary
+slide_dict = ppu.load_time_slides(trig_file)
+
+# Generate segments dictionary
+segment_dict = ppu.load_segment_dict(trig_file)
+
+# Construct trials removing vetoed times
+trial_dict, total_trials = ppu.construct_trials(
+    opts.seg_files,
+    segment_dict,
+    ifos,
+    slide_dict,
+    opts.veto_file
+)
+
+# Load trigger and injections data: ensure that newtwork power chi-square plots
+# show all the data to see the impact of the reweighted SNR cut, otherwise remove
+# points with reweighted SNR below threshold
+rw_snr_threshold = None if veto_type == 'network' else opts.newsnr_threshold
+trig_data = ppu.load_data(trig_file, ifos, data_tag='trigs',
+                          rw_snr_threshold=rw_snr_threshold,
+                          slide_id=opts.slide_id)
+inj_data = ppu.load_data(found_missed_file, ifos, data_tag='injs',
+                         rw_snr_threshold=rw_snr_threshold,
+                         slide_id=0)
+
+# Dataset name for the horizontal direction
+if snr_type == 'single':
+    x_key = ifo + '/snr'
+else:
+    x_key = 'network/' + snr_type + '_snr'
+# Dataset name for the vertical direction and for normalization
+if veto_type == 'power':
+    y_key = opts.ifo + '/chisq'
+elif veto_type in ['bank', 'auto']:
+    y_key = opts.ifo + '/' + veto_type +'_chisq'
+else:
+    y_key = 'network/my_network_chisq'
+
+keys = [x_key, y_key]
+# The network chi-square is already normalized so dof_key is not needed
+if veto_type != 'network':
+    dof_key = y_key + '_dof'
+    keys += [dof_key]
```

Comment:
Looks like `dof_key` is not used anywhere else, so I would avoid defining `dof_key` and turn these lines into a single line.